src/components/QuizReview/QuizReviewSummary.jsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Divider, Typography } from '@mui/material';
+import useQuizContext from '../../hooks/useQuizContext';
+
+function QuizReviewSummary() {
+  const { quizState } = useQuizContext();
+
+  return (
+    <div>
+      {quizState.questions.map((question, index) => (
+        <Box key={index} sx={{ mb: 2 }}>
```

Comment:
Never use indexes as keys

Reply:
Nope it is not, the thing is that if the array element is removed/moved/added in the middle the keys will stay the same but the element itself is changed, this means that React may not "see" that the component needs to be rendered because the cay is the same. 

This is a noobie mistake that shows lack of experience in react, this kind of things have huge impact on your assessment on the job interview.

Please get yourself familiar with the docs about keys https://react.dev/learn/rendering-lists#where-to-get-your-key